### DISCUSSION TITLE
Migrate Webbrowser control to CEFSharp

### DISCUSSION THREAD

**Discussion by Ammertos** [cefsharp/CefSharp#4437] [General]
Hey,

I'm fairly new to this but I want to migrate the old MS webbrowser control to CEFSharp.

my code looks like this (see below)

How would the line :             

webControl.Navigate(providerUrl, null, postdata, "Content-Type: application/xml" + Environment.NewLine + "Content-Type: " + contentType + Environment.NewLine);

look in CEFSharp? 

Any help would be appreciated.

private void CallHttpWebsite()
        {
            tb_Link.Text = providerUrl;

            byte[] postdata = null;

            if (postData == PostDataEnum.FormData)
            {
                postdata = SetFormDataPost(targetUrl, saml, out contentType);
            }
            else
            {
                postdata = SetUrlEncodedPost(targetUrl, saml, out contentType);
            }

            //providerUrl = "http://www.google.com";
            
            webControl.Navigate(providerUrl, null, postdata, "Content-Type: application/xml" + Environment.NewLine + "Content-Type: " + contentType + Environment.NewLine);
        }

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#load-url-with-postdata
